src/main/scala/GuessWhoGame/Board.scala

```diff
@@ -1,7 +1,116 @@
 package GuessWhoGame
 
-case class Board (Characters: List[Character]) {
+import scala.util.Random
+
+//values could be more domain specific, name/selecting could be within character case class, then able to test separately
+case class Board(characters: List[Character]) {
   def printCharacterNames(): Unit = {
-    Characters.foreach(Character => println(Character.name))
+    characters.foreach(Character => println(Character.name))
+  }
+
+  val selectedCharacter: Character = selectRandomCharacter(characters)
+  var remainingCharacters: List[Character] = characters
+
+  def printRemainingCharacters(): Unit = {
+    remainingCharacters.foreach(Character => println(Character.name))
+  }
+
+  def selectRandomCharacter(characters: List[Character]): Character = {
+    characters(Random.nextInt(characters.size))
+  }
+
+  //Testing purposes
+  def getSelectedCharacter: Character = selectedCharacter
+
+  def printSelectedCharacter(): Unit = {
+    println(selectedCharacter.name)
+  }
+
+  def getRemainingCharacters: List[Character] = remainingCharacters
```

Comment:
Nice use of inline notation (no {}) for a single line method!

Reply:
Thank you!